data/crac-file/crac-impl/src/test/java/com/farao_community/farao/data/crac_impl/threshold/AbsoluteFlowThresholdTest.java

```diff
@@ -101,23 +155,30 @@ public void computeMarginNoData() throws Exception {
 
     @Test
     public void computeMarginDisconnectedLine() throws Exception {
-        assertEquals(500 - 770, absoluteFlowThresholdAmps.computeMargin(networkWithtLf, cnec2), 1); // on cnec2, after LF: 770 A
+        // terminal 1 disconnected
+        networkWithtLf.getBranch("FRANCE_BELGIUM_2").getTerminal1().disconnect();
+        assertEquals(500.0 - 0.0, absoluteFlowThresholdAmps.computeMargin(networkWithtLf, cnec3), DOUBLE_TOL);
+        // terminal 2 disconnected
+        networkWithtLf.getBranch("FRANCE_BELGIUM_2").getTerminal1().connect();
+        networkWithtLf.getBranch("FRANCE_BELGIUM_2").getTerminal2().disconnect();
+        assertEquals(500.0 - 0.0, absoluteFlowThresholdAmps.computeMargin(networkWithtLf, cnec3), DOUBLE_TOL);
+        // both terminal disconnected
         networkWithtLf.getBranch("FRANCE_BELGIUM_2").getTerminal1().disconnect();
-        assertEquals(500, absoluteFlowThresholdAmps.computeMargin(networkWithtLf, cnec2), 1);
+        assertEquals(500.0 - 0.0, absoluteFlowThresholdAmps.computeMargin(networkWithtLf, cnec3), DOUBLE_TOL);
     }
 
     @Test
     public void synchronize() {
-        assertEquals(500, absoluteFlowThresholdAmps.getMaxValue(), 1);
+        assertEquals(500.0, absoluteFlowThresholdAmps.getMaxValue(), 1);
         cnec1.synchronize(networkWithoutLf);
-        assertEquals(500, absoluteFlowThresholdAmps.getMaxValue(), 1);
+        assertEquals(500.0, absoluteFlowThresholdAmps.getMaxValue(), 1);
     }
```

Comment:
Do we need this test?

Reply:
just to check that synchronisation does not change AbsoluteFlowThreshold.getMaxValue()